app/uk/gov/hmrc/exports/models/Pointer.scala

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmrc.exports.models
+
+import uk.gov.hmrc.exports.models.PointerSectionType.WCOPointerSectionType
+
+case class Pointer(sections: Seq[PointerSection]) {
```

Comment:
This will represent the entire pointer in the XML (multiple `Pointer` xml nodes).  The existing model models it as `Seq[ErrorPointer]`.  
I think the existing model will be awkward to work with in terms of the mapping as we're going to need a tonne of helper methods and the knowledge of wether it's a sequence element or not